### PR TITLE
addEven function works - D4rth

### PR DESCRIPTION
**Pull request by Morfiniusz**
:)

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: add-even
🏅 Granted 5 XP for @Morfiniusz

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Morfiniusz for on-time delivery

File: homework/add-even/addEven.hpp [hunk 1]
```diff
@@ -2,7 +2,12 @@
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+    int sum = 0;
+    for(const auto& num : numbers){
+        if(num%2 == 0){
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:7]: Add space between operations like '+' '-' '!=' '==' '%' etc...

File: homework/add-even/addEven.hpp [hunk 2]
```diff
@@ -2,7 +2,12 @@
...
+            sum = sum + num;
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:8]: sum += num;

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**